wfrc_example/utah_bike_demand_model.py

```diff
@@ -54,7 +54,7 @@ def main():
 
 
 @preprocessor()
-def preprocess_network(net):
+def preprocess_network(net, settings):
```

Comment:
preprocess_network now gets settings so that the generalized cost function is parameterizable